public/homepage/scripts/glitch.js

```diff
@@ -0,0 +1,14 @@
+/* globals $: true */
+
+var $ = require("jquery");
+
+module.exports = {
+  init: function() {
+    let cta = $(".glitch-cta.underlay");
+    if (cta) {
+      setTimeout(() => cta.removeClass("hidden"), 2000);
```

Comment:
can we change this to `1000`?

Reply:
I tried that, but this causes the dialog to pop up even before all the content has visually loaded, which is an odd experience.